scripts/dynamicHtml.js

```diff
@@ -125,7 +154,14 @@ function newImageUpload(uid) {
         <button class="btn btn-info font-weight-bold" id="viewFinalImgBtn_${uid}">Converted</button>
       </div>
     </div>
-    <button class="btn btn-warning mr-sm-3" id="imgEditBtn_${uid}">Edit</button>
+    <div class="btn-group">
+      <button class="btn btn-info" id="saveFormDataBtn_${uid}">💾 Save
+      <a data-toggle="tooltip" data-placement="top" data-html="true" title="Save to the browser's local storage.
+        No data is sent to the server. Has limited storage capacity. Delete other images if you cannot save anymore."
```

Comment:
Pending test/fix: I did not yet test what happens when local storage is exceeded.

Reply:
Tested now, and added an alert if that happens.